presidio-analyzer/tests/test_analyzer_engine_provider.py

```diff
@@ -0,0 +1,71 @@
+import re
+from pathlib import Path
+
+from presidio_analyzer import AnalyzerEngine, AnalyzerEngineProvider
+from presidio_analyzer.nlp_engine import SpacyNlpEngine
+
+from presidio_analyzer.nlp_engine.transformers_nlp_engine import TransformersNlpEngine
+
+def assert_default_configuration(engine: AnalyzerEngine):
+    assert engine.supported_languages == ["en"]
+    assert engine.registry.global_regex_flags == re.DOTALL | re.MULTILINE | re.IGNORECASE
+    assert engine.default_score_threshold == 0
+    assert len(engine.registry.recognizers) == 29
+    names = [recognizer.name for recognizer in engine.registry.recognizers]
+    assert set(names) == set(["UsBankRecognizer", "UsLicenseRecognizer", "UsItinRecognizer", 
```

Comment:
Recognizers change all the time, it's better not to have a static list of recognizers here

Reply:
The thing is we're now introducing default configuration. I thought it would make sense to break a test in case something over there changes.